qark/qark.py

```diff
@@ -1,6 +1,55 @@
-def main():
-    pass
+from __future__ import absolute_import
+
+import logging
+
+import click
+
+from qark.decompiler.decompiler import Decompiler
+from qark.scanner.scanner import Scanner
+from qark.report import Report
+
+log = logging.getLogger(__name__)
 
 
-if __name__ == "__main__":
-    main()
+@click.command()
+@click.option("--sdk-path", type=click.Path(exists=True, file_okay=False, resolve_path=True),
+              help="Path to the downloaded SDK directory if already downloaded")
+@click.option("--build-path", type=click.Path(resolve_path=True, file_okay=False),
+              help="Path to place decompiled files and exploit APK", default="build", show_default=True)
+@click.option("--debug/--no-debug", default=False, help="Show debugging statements (helpful for issues)",
+              show_default=True)
+@click.option("--apk", "source", help="APK to decompile and run static analysis. If passed, "
+                                      "the --java option is not used",
+              type=click.Path(exists=True, resolve_path=True, file_okay=True, dir_okay=True))
+@click.option("--java", "source", type=click.Path(exists=True, resolve_path=True, file_okay=True, dir_okay=True),
+              help="A directory containing Java code, or a Java file, to decompile and run static analysis. If passed,"
+                   "the --apk option is not used")
+@click.option("--report-type", type=click.Choice(["html", "xml", "json", "csv"]),
+              help="Type of report to generate along with terminal output", default="html", show_default=True)
+@click.option("--exploit-apk/--no-exploit-apk", default=False,
```

Comment:
Add a flag for the location of the java executable maybe?

Reply:
I created an issue for this #178 since it isn't pressing right now. It is also available for someone else to contribute.